identity/app/views/fragments/socialSignin.scala.html

```diff
@@ -21,4 +21,13 @@
             @fragments.inlineSvg("share-google", "icon", List("social-signin__icon", "social-signin__icon--google"))
         </a>
     </li>
+    <li class="social-signin__item">
+        <a class="social-signin__action social-signin__action--apple"
+        data-link-name="Register/Sign in with Apple" data-test-id="apple-sign-in"
+        href="@idUrlBuilder.buildOAuthUrl("/apple/" + route.apple.path, idRequest, ThirdPartyConditions.optParams(idRequest, idUrlBuilder))
```

Comment:
does route.apple.path need to be set?

Reply:
well a value has to be set, but all 3 oauth providers use the same path (`confirm`  in this case), so really the same could be used for all of them.
I just stuck to the convention that was already there